README.md

```diff
@@ -1 +1,52 @@
-# Sweetsour Parser
+<p align="center">
+  <img alt="Sweetsour" src="/docs/logo.png?raw=true" width="200px" />
+</p>
+<h1 align="center">Sweetsour</h1>
+<p align="center">
+  A CSS-in-JS parser and pipeline adhering to the ISTF spec 🍭
+</p>
+
+<br />
+
+This is a reference implementation of a CSS-in-JS parser, outputting ISTF, according
+to the current [ISTF specification](https://github.com/cssinjs/istf-spec).
+
+## Status
+
+**Work in Progress: Not ready for any use!**
+
+Currently work is ongoing to build: **The parser (~60%)**
+
+- [ ] Parser
+- [ ] Transform tool with plugin support
+- [ ] Reference Babel plugin
+- [ ] Flattener for nested CSS (plugin)
+- [ ] Static prefixer (plugin)
+
+## Goals
+
+The project aims to build a common and reusuable pipeline for processing CSS, including
+interleaved interpolations, for CSS-in-JS libraries; This involves processing CSS-in-JS code,
+turning it into ISTF, and to provide a pipeline to then transform this code during either
+the build time (preprocessing), or runtime (postprocessing).
+
+The goals of ISTF and Sweetsour include:
+
+- Providing a common CSS-in-JS pipeline
```

Comment:
Those are evtl separate goals for different projects/packages.

Reply:
True; I’ll modify it to be Sweetsour centric for now